cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -98,6 +120,18 @@ function create-resource-from-string() {
 # managed result is of that. Start everything below that directory.
 echo "== Kubernetes addon manager started at $(date -Is) =="
 
+# Load the kube-env, which has all the environment variables we care
+# about, in a flat yaml format.
+kube_env_yaml="/var/cache/kubernetes-install/kube_env.yaml"
```

Comment:
@justinsb this should be a no-op if this file doesn't exist, but since I believe this runs on AWS as well cc'ing you as an FYI.